### PR TITLE
ITNFA-11: Fix national identifier generation

### PR DESCRIPTION
**Pull request by pgesek**

### CODE REVIEW

File: api/src/test/java/org/openmrs/module/m2sysbiometrics/service/RegistrationServiceTest.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void shouldTryToRegisterNationallyWithoutException() throws Exception {
         when(nationalBioServerClient.enroll(subject.getSubjectId(), capture.getTemplateData())).thenReturn(EXISTING_RESULT_XML);
 
         //when
-        registrationService.registerNationally(subject, capture);
+        registrationService.registerNationally(subject.getSubjectId(), capture);
```
Comment (pawelcieszko) [inline api/src/test/java/org/openmrs/module/m2sysbiometrics/service/RegistrationServiceTest.java]: I would define a new national guid to emphasize that this it is not related to subjectid

**Review by pawelcieszko — CHANGES_REQUESTED** [review]

File: api/src/test/java/org/openmrs/module/m2sysbiometrics/service/RegistrationServiceTest.java (hunk 1 above)
Comment (pgesek) [reply api/src/test/java/org/openmrs/module/m2sysbiometrics/service/RegistrationServiceTest.java]: Ok I will, got a bit lazy in this test

**Comment by pawelcieszko** [comment]
please resolve the conflicts

**Review by pawelcieszko — APPROVED** [review]
